atlasdb-ete-tests/Dockerfile

```diff
@@ -1,6 +1,23 @@
-FROM frolvlad/alpine-oraclejdk8:slim
-
-RUN apk update && apk add libstdc++ bash wget
+FROM centos:centos7
+
+MAINTAINER hsaraogi@palantir.com <Himangi Saraogi>
+
+# Default to UTF-8 file.encoding
+ENV LC_ALL en_US.UTF-8
+
+ENV JAVA_HOME /usr/java/jdk1.8.0_102/jre
+ENV JAVA_VERSION 8u102
+ENV RPM_VERSION 8u102-b14
+
+# - install java8 jdk
+RUN yum install -y wget && \
+    wget \
+      --no-cookies \
+      --no-check-certificate \
+      --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" \
+      "http://download.oracle.com/otn-pub/java/jdk/${RPM_VERSION}/jdk-${JAVA_VERSION}-linux-x64.rpm" && \
+    yum localinstall -y "jdk-${JAVA_VERSION}-linux-x64.rpm" && \
+    rm -f "jdk-${JAVA_VERSION}-linux-x64.rpm"
 
 ENV DOCKERIZE_VERSION v0.2.0
 
```

Comment:
@tpetracca This particular Alpine image doesn't have the right version of `ps`. Also, the internal spec is designed to support only centos systems. I think it makes sense to test with centos owing to this, but we might be able to get around this by installing the right version of `ps` on the alpine container. @mnazbro or SA.